src/umpire/strategy/DynamicPoolList.hpp

```diff
@@ -34,8 +34,17 @@ namespace strategy {
  */
 class DynamicPoolList : public AllocationStrategy {
  public:
+  /*!
+   * \brief Coalescing Heuristic functions for Percent-Releasable and Blocks-Releasable. Both have
+   * the option to reallocate to High Watermark instead of actual size of the pool (actual size is
+   * currently the default).
+   */
   static PoolCoalesceHeuristic<DynamicPoolList> percent_releasable(int percentage);
+  static PoolCoalesceHeuristic<DynamicPoolList> percent_releasable_hwm(int percentage);
```

Comment:
Should we combine these two functions by using default parameters?

```suggestion
  static PoolCoalesceHeuristic<DynamicPoolList> percent_releasable(int percentage, bool resize_to_hwm = false);
```

Reply:
As per our 11/22 Umpire meeting, we decided this needed more discussion. Having the two separate functions like that is kind of "self documenting" in a nice way, but then again won't scale well if we decide further down the line to have some other size to coalesce to. So, we may need to figure something else out and the fact that we are dealing with lambdas in these functions makes it tough. So: I will resolve this conversation for now, but will open a JIRA ticket for further discussion.

Reply:
See https://rzlc.llnl.gov/jira/browse/UM-1022